src/transactions/feed/queryHelper.ts

```diff
@@ -282,31 +304,45 @@ async function queryTransactionsFeed({
   address,
   localCurrencyCode,
   params,
+  updateTransactionState,
+  setActiveRequests,
+  activeRequests,
 }: {
   address: string | null
   localCurrencyCode: string
   params: Array<{
     networkId: NetworkId
     afterCursor?: string
   }>
-}): Promise<{ [key in NetworkId]?: QueryResponse }> {
-  const results = await Promise.all(
-    params.map(({ networkId, afterCursor }) =>
-      queryChainTransactionsFeed({
+  updateTransactionState: (networkId: NetworkId, data: QueryResponse | null) => void
```

Comment:
nit: to make it clear it's a callback.
```suggestion
  onNetworkResponse: (networkId: NetworkId, data: QueryResponse | null) => void
```